src/main/resources/GoogleApps/Translations.fr.xml

```diff
@@ -39,17 +39,27 @@
-googleapps.config.explanation=XWiki peut-etre intégré avec Google Apps, ce qui permet ensuite l"édition des attachements Office en utilisant les éditeurs Google Apps ainsi que leur sauvegarde dans le wiki. Cela permet également de télécharger un fichier Google Apps dans n"importe quelle page du Wiki. Afin d'activer Google Apps vous devez vous [[enregistrer pour l'accès à OAuth sur Google.com&gt;&gt;https://accounts.google.com/ManageDomains]] et insérer votre nom de domaine et la clé dans les paramètres suivants.
+googleapps.config.explanation=XWiki peut-etre intégré avec Google Apps, ce qui permet ensuite l'édition des attachements Office en utilisant les éditeurs Google Apps ainsi que leur sauvegarde dans le wiki. Cela permet également de télécharger un fichier Google Apps dans n'importe quelle page du Wiki. Afin d'activer Google Apps vous devez vous [[enregistrer pour l'accès à OAuth sur Google.com>>https://accounts.google.com/ManageDomains]] et insérer les informations dans les paramètres suivants (voir aussi  please see [[les instructions d'installation>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]]).
+GoogleApps.GoogleAppsConfigClass_active=Activer
 GoogleApps.GoogleAppsConfigClass_appname=Nom de l'application
-GoogleApps.GoogleAppsConfigClass_key=Clé privée
+GoogleApps.GoogleAppsConfigClass_appname.hint=Pour votre propre organisation.
+GoogleApps.GoogleAppsConfigClass_clientid=ID client
+GoogleApps.GoogleAppsConfigClass_clientid.hint=Indiqué dans la console Google.
+GoogleApps.GoogleAppsConfigClass_secret=Code secret du client
+GoogleApps.GoogleAppsConfigClass_secret.hint=Indiqué dans la console Google.
 GoogleApps.GoogleAppsConfigClass_domain=Domaine Google Apps
-GoogleApps.GoogleAppsConfigClass_active=Activer
-GoogleApps.GoogleAppsConfigClass_useCookies=Permettre l'usage des cookies
-GoogleApps.GoogleAppsConfigClass_skipLoginPage=(Il est nécessaire que les cookies soient permis) Sauter la page d'autentification
-GoogleApps.GoogleAppsConfigClass_authWithCookies=(Il est nécessaire que les cookies soient permis) Permettre l'autentification avec cookie - sauter google si déjà connecté une fois
-GoogleApps.GoogleAppsConfigClass_cookiesTTL=(Il est nécessaire que les cookies soient permis) Durée de vie des cookies en secondes - défault : 3 jours.
+GoogleApps.GoogleAppsConfigClass_domain.hint=Restreint le domaine des utilisateurs autorisés à s'authentifier sur le wiki.
+GoogleApps.GoogleAppsConfigClass_useCookies=Utiliser les cookies
+GoogleApps.GoogleAppsConfigClass_useCookies.hint=Ces cookies permettent au navigateur de mémoriser quel compte Google était utilisé pour s'authentifier sur le wiki.
+GoogleApps.GoogleAppsConfigClass_skipLoginPage=(cookies nécessaires) Sauter la page d'autentification
+GoogleApps.GoogleAppsConfigClass_skipLoginPage.hint=Permettre aux navigateurs ayant le cookie de s'authentifier automatiquement (fonction <i>se souvenir de moi</i>).
```

Comment:
We should only use plain text and to not allow users to change `<i>` with `<b>`, for example. Parametrized translation should fix this.